bot/gif_command_test.go

```diff
@@ -12,46 +12,62 @@ import (
 )
 
 func TestGIF(t *testing.T) {
-	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		assert.Equal(t, "GET", r.Method)
-		assert.Equal(t, "/v1/gifs/search", r.URL.Path)
+	cases := []struct {
```

Comment:
so, the only thing we need to vary between test cases is the explicit flag being set or not; in that case, we could have our `cases` just be `[]bool{ true, false }`; but a little bit more polished might be:
```
cases := map[string]bool {
  "explicit flag disabled: false,
  "explicit flag enabled": true,
}

for name, explicit := range cases {
    t.Run(name, func(t *testing.T){ 
    ...
}
```

Then,  each case would have everything else the same, e.g. "dogs playing poker", key, url, etc.